general/management/commands/gm_worker_async_tasks.py

```diff
@@ -90,3 +92,17 @@ def task_whitelist_user(self, gearman_worker, gearman_job):
                 count_done = count_done + 1
             self.write_stdout("Finished processing one ticket, %d remaining" % (len(tickets)-count_done))
         return 'true' if len(tickets) == count_done else 'false'
+
+    def task_email_random_sound(self, gearman_worker, gearman_job):
+        self.write_stdout("Notifying user of random sound of the day")
```

Comment:
Are we happy to send an email to any user regardless of how old the sound is? I guess the idea here is to engage people, including those who haven't used FS for a while.
Some situations which might happen:
 - user's email doesn't work any more
 - sound was uploaded 5 years ago and the user forgot that they even added it
 - the random sound is a goodsounds track, or something from one of the sample libraries [congratulations, one of your 4000 sounds has been chosen as random. F# is a great note!]

I realise that this means considering how the random sound selection works. Are we happy with how it currently works?